extensions/wikia/AchievementsII/AchConfig.class.php

```diff
@@ -346,6 +351,14 @@ public function isSecret($badgeTypeId) {
 		return in_array($badgeTypeId, self::$mSecretBadges);
 	}
 
+	public function shouldShow($badgeTypeId) {
+		global $wgAchievementsEditAddPhotoOnly;
+		if (empty($wgAchievementsEditAddPhotoOnly) || in_array($badgeTypeId, self::$mEditAddphotoOnlyBadges)) {
```

Comment:
This should all be spacey per our PHP coding conventions. Same throughout the changes.